webapp/src/main/java/com/box/l10n/mojito/pagerduty/PagerDutyIntegrations.java

```diff
@@ -0,0 +1,41 @@
+package com.box.l10n.mojito.pagerduty;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.stereotype.Component;
+
+@Component
+@DependsOn("pagerDutyIntegrationsConfiguration")
+public class PagerDutyIntegrations {
```

Comment:
I personally think class names should stay singular since they usually represent a single object. So I a name like `PagerDutyIntegrationService` makes more sense to me, because it is a single service

Reply:
I agree, will update the name, thank you!